gocourse04/hw/editor/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+func scanner(description string) []string {
+	var text []string
+	scanner := bufio.NewScanner(os.Stdin)
+	fmt.Printf(description)
+	for scanner.Scan() {
+		line := scanner.Text()
+		if line == "" {
```

Comment:
in theory, this shouldn't happen. unless you have some input that looks like
```
asdsd
asds

kjdshdfkj
```
but in that case I think we should still try to handle it and not break the loop (scanner should do that automatically)

Reply:
ok, nevermind, I see that you use two different inputs that you are splitting by a newline